From 5a983850d93172339ac48c82497d127d01d40fd6 Mon Sep 17 00:00:00 2001
From: Christian Bruel <christian.bruel@foss.st.com>
Date: Mon, 11 Dec 2023 14:18:57 +0100
Subject: [PATCH] arm64: dts: st: add a sleep pinctrl for pcie on
 stm32mp25-pinctrl.dtsi

Add Analog pins of PCIe to perform power cycle

Signed-off-by: Christian Bruel <christian.bruel@foss.st.com>
Change-Id: I9bb022a211e819b602e3aff8ed450a9b70511ee5
---
 arch/arm64/boot/dts/st/stm32mp25-pinctrl.dtsi | 6 ++++++
 1 file changed, 6 insertions(+)

--- a/arch/arm64/boot/dts/st/stm32mp25-pinctrl.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp25-pinctrl.dtsi
@@ -413,6 +413,12 @@
 		};
 	};
 
+	pcie_sleep_pins_a: pcie-sleep-0 {
+		pins {
+			pinmux = <STM32_PINMUX('J', 0, ANALOG)>;
+		};
+	};
+
 	rtc_out2_rmp_pins_a: rtc-out2-rmp-pins-0 {
 		pins {
 			pinmux = <STM32_PINMUX('I', 8, ANALOG)>; /* RTC_OUT2_RMP */
